src/colorNamer.js

```diff
@@ -117,30 +117,30 @@ var oghsv;
 
 var xcp = 
 [
-  {
-    "h":0,
-    "s":0,
-    "b":0.8275,
-    "name":"gray"
-  },
-  {
-    "h":0,
-    "s":0,
-    "b":0.8627,
-    "name":"gray"
-  },
-  {
-    "h":0,
-    "s":0,
-    "b":0.7529,
-    "name":"gray"
-  },
-  {
-    "h":0.0167,
-    "s":0.1176,
-    "b":1,
-    "name":"light pink"
-  },
+	{
+		"h":0,
+		"s":0,
```

Comment:
oo same tab thing here

Reply:
same thing! lets choose one and I'll change colornamer

Reply:
@lm-n 1 tab = 2spaces?

Reply:
sounds good

Reply:
I have an atom setup that lints on save if that helps!